src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -60,33 +57,22 @@ internal sealed partial class ConsoleHost
         // NTRAID#Windows Out Of Band Releases-915506-2005/09/09
         // Removed HandleUnexpectedExceptions infrastructure
         /// <summary>
-        ///
         /// internal Entry point in msh console host implementation
-        ///
         /// </summary>
-        ///
         /// <param name="bannerText">
         /// Banner text to be displayed by ConsoleHost
         /// </param>
-        ///
         /// <param name="helpText">
         /// Help text for minishell. This is displayed on 'minishell -?'.
         /// </param>
-        ///
         /// <param name = "args">
-        ///
         /// Command line parameters to pwsh.exe
-        ///
         /// </param>
         /// <returns>
-        ///
         /// The exit code for the shell.
-        ///
         /// NTRAID#Windows OS Bugs-1036968-2005/01/20-sburns The behavior here is related to monitor work.  The low word of the
```

Comment:
Remove "NTRAID#Windows OS Bugs-1036968-2005/01/20-sburns ".  NTRAID is a really old bug tracking system that is no longer running so no way to open this bug.

Reply:
Removed.